Out-source build fails

From a fresh clone:
```
git clone --recursive git@github.com:votca/votca
cd votca
mkdir build
cd build
cmake -DBUILD_CTP=ON ..
make
...
[ 87%] Building CXX object ctp/src/libctp/CMakeFiles/votca_ctp.dir/version.cc.o
/home/christoph/votca/src/votca/ctp/src/libctp/version.cc:36:10: fatal error: gitversion.h: No such file or directory
 #include "gitversion.h"
          ^~~~~~~~~~~~~~
compilation terminated.
```

--- a/src/libctp/CMakeLists.txt
+++ b/src/libctp/CMakeLists.txt
@@ -1,11 +1,11 @@
 
 #configure_file(votca_config.h.in ${CMAKE_CURRENT_BINARY_DIR}/votca_config.h)
-#include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 add_custom_target(gitversion-ctp COMMAND ${CMAKE_COMMAND} -DTOP_SOURCE_DIR="${CMAKE_SOURCE_DIR}"
   -DGIT_EXECUTABLE="${GIT_EXECUTABLE}"
   -P ${CMAKE_MODULE_PATH}/gitversion.cmake)
 set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES gitversion.h)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 
 file(GLOB_RECURSE VOTCA_SOURCES *.cc)

